msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md

```diff
@@ -285,73 +285,112 @@ A simple interface for actions and predictions allows bots to react when they ha
 
 Thanks to our AI library, the prompt needs only to outline the actions supported by the bot, and supply a few-shot examples of how to employ those actions. Conversation history helps with a natural dialogue between the user and bot, such as *add cereal to groceries list*, followed by *also add coffee*, which should indicate that coffee is to be added to the groceries list.
 
-All entities are required parameters to actions.
+The following is a conversation with an AI assistant. The assistant can manage lists and supports the following commands:
```

Comment:
The AI assistant is capable of managing lists and recognizes the subsequent commands:

Reply:
Updated.